src/helpers/validator.js

```diff
@@ -2,11 +2,14 @@ import Joi from 'joi';
 
 const Schema = {
   login: Joi.object().keys({
-    email: Joi.string().required().email().message('Please fill all fields'),
+    email: Joi.string()
+      .required()
+      .email()
+      .error(new Joi.ValidationError('Please fill all fields')),
```

Comment:
I suggest you leave the default error message, since the ones you are adding might not cover all cases, eg invalid email format case